src/index.css

```diff
@@ -1,10 +1,9 @@
 body {
```

Comment:
We could move these global styles to styled components: https://styled-components.com/docs/api#createglobalstyle
That way, we'll be able to use our theme easily for global styles. Of course, this is out of scope of this PR - I added a separate task in Jira: https://dareit.atlassian.net/browse/DRM-35

Reply:
Ok. I can do it. :)